### DISCUSSION TITLE
Hover tooltip remain open when hovering over textbox

### DISCUSSION THREAD

**Discussion by catherineymgui** [grafana/grafana#57636] [Saga Design System]
Hi team,

Something I noticed is that the hover tooltip disappears as soon as someone moves their cursor away from the info icon. This is slightly annoying if I move my cursor towards the textbox and it disappears, but also leads to cases where there can't be a link the tooltip, but just has copy to "see documentation"

https://user-images.githubusercontent.com/5264102/197880774-1198481f-3652-4375-8d02-4d605a3b9145.mov

1. Are there any best practices around linking to another page in a tooltip?
2. Would it be possible to have a hover tooltip remain open as long when someone is hovering over the text box as well?

**Comment by staton-hyse11** [comment]
1. Moving forward, links in tooltips is not a recommended pattern. There is a new component being created soon that will allow for interactive content. See: [Toggletip](https://github.com/grafana/grafana/issues/56032)
2. Having the tooltip remain active on hover of the tooltip body is currently a discussion point in our [tooltip](https://github.com/grafana/grafana/issues/56031) documentation
